content/zh/docs/concepts/workloads/pods/pod-lifecycle.md

```diff
@@ -1,180 +1,829 @@
+or is [terminated](#pod-termination).
+-->
+在 Kubernetes API 中，Pod 包含规约部分和实际状态部分。
+Pod 对象的状态包含了一组 [Pod 状况（Conditions）](#pod-conditions)。
+如果应用需要的话，你也可以向其中注入[自定义的就绪性信息](#pod-readiness-gate)。
 
+Pod 在其生命周期中只会被[调度](/zh/docs/concepts/scheduling-eviction/)一次。
+一旦 Pod 被调度（分派）到某个节点，Pod 会一直在该节点运行，直到 Pod 停止或者
+被[终止](#pod-termination)。
 
 <!-- body -->
 
+<!--
+## Pod lifetime
+
+Like individual application containers, Pods are considered to be relatively
+ephemeral (rather than durable) entities. Pods are created, assigned a unique
+ID ([UID](/docs/concepts/overview/working-with-objects/names/#uids)), and scheduled
+to nodes where they remain until termination (according to restart policy) or
+deletion.  
+If a {{< glossary_tooltip term_id="node" >}} dies, the Pods scheduled to that node
+are [scheduled for deletion](#pod-garbage-collection) after a timeout period.
+-->
+## Pod 生命期   {#pod-lifetime}
+
+和一个个独立的应用容器一样，Pod 也被认为是相对临时性（而不是长期存在）的实体。
+Pod 会被创建、赋予一个唯一的
+ID（[UID](/zh/docs/concepts/overview/working-with-objects/names/#uids)），
+并被调度到节点，并在终止（根据重启策略）或删除之前一直运行在该节点。
+
+如果一个{{< glossary_tooltip text="节点" term_id="node" >}}死掉了，调度到该节点
+的 Pod 也被计划在给定超时期限结束后[删除](#pod-garbage-collection)。
+
+<!--
+Pods do not, by themselves, self-heal. If a Pod is scheduled to a
+{{< glossary_tooltip text="node" term_id="node" >}} that then fails,
+or if the scheduling operation itself fails, the Pod is deleted; likewise, a Pod won't
+survive an eviction due to a lack of resources or Node maintenance. Kubernetes uses a
+higher-level abstraction, called a
+{{< glossary_tooltip term_id="controller" text="controller" >}}, that handles the work of
+managing the relatively disposable Pod instances.
+-->
+Pod 自身不具有自愈能力。如果 Pod 被调度到某{{< glossary_tooltip text="节点" term_id="node" >}}
+而该节点之后失效，或者调度操作本身失效，Pod 会被删除；与此类似，Pod 无法在节点资源
+耗尽或者节点维护期间继续存活。Kubernetes 使用一种高级抽象，称作
+{{< glossary_tooltip term_id="controller" text="控制器" >}}，来管理这些相对而言
+可随时丢弃的 Pod 实例。
+
+<!--
+A given Pod (as defined by a UID) is never "rescheduled" to a different node; instead,
+that Pod can be replaced by a new, near-identical Pod, with even the same name i
+desired, but with a different UID.
+
+When something is said to have the same lifetime as a Pod, such as a
+{{< glossary_tooltip term_id="volume" text="volume" >}},
+that means that the thing exists as long as that specific Pod (with that exact UID)
+exists. If that Pod is deleted for any reason, and even if an identical replacement
+is created, the related thing (a volume, in this example) is also destroyed and
+created anew.
+-->
+任何给定的 Pod （由 UID 定义）从不会被“重新调度（rescheduled）”到不同的节点；
+相反，这一 Pod 可以被一个新的、几乎完全相同的 Pod 替换掉。
+如果需要，新 Pod 的名字可以不变，但是其 UID 会不同。
+
+如果某物声称其生命期与某 Pod 相同，例如存储{{< glossary_tooltip term_id="volume" text="卷" >}}，
+这就意味着该对象在此 Pod （UID 亦相同）存在期间也一直存在。
+如果 Pod 因为任何原因被删除，甚至某完全相同的替代 Pod 被创建时，
+这个相关的对象（例如这里的卷）也会被删除并重建。
+
+{{< figure src="/images/docs/pod.svg" title="Pod diagram" width="50%" >}}
```

Comment:
If you want, you can translate the title too.

Reply:
good catch.